schemas/2.2/schema.json

```
@@ -31,6 +31,11 @@
           "description": "If supported by the software the administrator configured name of this instance",
           "type": "string",
           "pattern": "^.{0,500}$"
+        },
+        "description": {
+          "description": "If supported by the software the administrator configured long form description of this instance",
```

```suggestion
          "description": "If supported by the software the administrator configured long form description of this instance. It might be in Markdown format.",
```

---

@jhass ?

---

@jhass that's why I exactly propose an optional format in markdown (or as you are right this is not a standart commonmark). if you stricktly specify that only common mark might be here, normal text will show as normal text and you can block all other things that would allow xss - almost all commonmark/markdown libs allow to have a strickt mode and not allow html.

If you have an idea how to alter the description to make clear , that **html** is not used and will be consideret and **must** show as text - this should deny xss.

as we only describe a spec here we can not do more against xss as in the end it depends on the impl.

---

Is there a standard as widely adopted and easy to prevent XSS as MD ?

yes it sucks that it's security is opt-in not opt-out

---

```suggestion
          "description": "If supported by the software the administrator configured long form description of this instance. It might be in CommonMark format, HTML must not be rendered.",
```

and as for XSS, if we don't propose a format, admins can and so they still will do strange *shit*, with and without our proposal :D

---

@jhass yes/no ?

---

```suggestion
          "description": "If supported by the software the administrator configured long form description of this instance.",
```

As it is a blocker - I'll move it into it's own pull

---

->  #86